src/client/modules/navigation/index.js

```diff
@@ -0,0 +1,156 @@
+
+    .controller('NavigationCtrl', ['$scope', '$http', '$location', '$window', '$uibModal',
+        ($scope, $http, $location, $window, $uibModal) => {
+
+            $scope.logged = false;
+
+            $scope.items = [];
+
+            $scope.logInClick = () => {
+                const modalInstance = $uibModal.open({
+                    animation: true,
+                    templateUrl: 'login/login.html',
+                    controller: 'LoginCtrl',
+                    resolve: {
+                        items: () => {
+                            return $scope.items;
+                        }
+                    }
+                });
+                modalInstance.result.then((selectedItem) => {
+                    $scope.selected = selectedItem;
+                });
+            };
+
+            $scope.logOutClick = () => {
+                $http.post('api/users/logout').then((response) => {
+                    $window.location.reload();
+                });
+            };
+
+            $scope.checkSessionFunc = () => {
+                $http.get("/api/users/checkSession").then((res) => {
+                    if (res.data.status === 'success') {
+                        $scope.logged = true;
+                    } else if (res.data.status == 'error'){
+                        $scope.logged = false;
+                    }
+                }, () => {
+                    console.log('server error');
+                });
+            };
+
+            $scope.checkSessionFunc();
+
+    }])
+
+
+    .directive('navigation', (routeNavigation) => {
+        return {
+            restrict: "E",
+            replace: true,
+            templateUrl: "navigation/navigation.html",
+            controller: ($scope) => {
+                $scope.routes = routeNavigation.routes;
+                $scope.activeRoute = routeNavigation.activeRoute;
+            }
+        };
+    })
+
+    .directive('navLog', ($http, routeNavigation, navConstructor) => {
+        return {
+            template: () => {
+                return navConstructor.navBuild(true);
+            },
+            controller: 'NavigationCtrl'
+        };
+    })
+
+    .directive('navNoLog', ($http, routeNavigation, navConstructor) => {
+        return {
+            template: () => {
+                return navConstructor.navBuild(false);
+            },
+            controller: 'NavigationCtrl'
+        };
+    })
+
+    .service('navConstructor', function (routeNavigation) {
+        this.navBuild = (logged) => {
+
+            let href, name, ngClick = "", icon = "";
+            let nav = '<ul class="nav navbar-nav"> ',
+                navCenter = '<a href="/game" class="btn btn-danger navbar-center">Start GAME</a>',
+                navRight = '<ul class="nav navbar-nav navbar-right"> ';
+            if(logged) {
+                for(let i = 0; i < routeNavigation.routes.length; i++) {
+                    href = ' href="' + routeNavigation.routes[i].template + '"';
```

Comment:
use es6 template literals, code will be cleaner 
https://developers.google.com/web/updates/2015/01/ES6-Template-Strings